### PR TITLE
Isolate dev code

### PR DESCRIPTION
**Pull request by kieferm**

### CODE REVIEW

File: local-dev.html [hunk 1]
```diff
@@ -12,16 +13,13 @@
 </head>
 <body>
     <p>
-        <button type="button" id="loadAFconfig">Load af database + server</button>
-        <button type="button" id="elementPath">Load af database + server</button>
-        <button type="button" id="getEFs">get EFs off element</button>
-        <button type="button" id="getTemplateAttributes">get attributes</button>
-        <button type="button" id="buildTreemap">Build treemap</button>
+        <button type="button" id="btnUpdate">Update</button>
```
Comment (jrsconfitto) [inline local-dev.html:16]: Thanks for removing all these buttons! (They were good at the time, but it's nice to see them gone)

**Review by jrsconfitto — CHANGES_REQUESTED** [review]
This is a helpful change, thanks. LGTM!

i wonder if those functions will work in product, but it looks to me like they don't even get called!

Please merge with master to pickup latest changes (and resolve subsequent conflicts), then i think it's good to go 😄

**Comment by jrsconfitto** [comment]
i'm a terrible reviewer! treemap.css is missing... @kieferm mind adding it in master?

**Comment by kieferm** [comment]
I think i had that file in another branch that was not yet merged.  In either case I just added it to master.

**Pull request merged**